bodhi/client/bindings.py

```diff
@@ -230,7 +222,7 @@ def query(self, **kwargs):
             # update ID, so try and figure it out
             if re.search(r'(\.el|\.fc)\d\d?', kwargs['package']):
                 kwargs['builds'] = kwargs['package']
-            elif re.search(r'FEDORA-(EPEL)?-\d{4,4}', kwargs['package']):
+            elif re.search(r'FEDORA-(EPEL-)?\d{4,4}', kwargs['package']):
```

Comment:
Hahahaha, well I guess we'll have to go back to the drawing board, but only if humans still exist then…